react/components/SearchResult.js

```diff
@@ -15,14 +15,14 @@ class SearchResult extends Component {
   static propTypes = searchResultPropTypes
 
   static defaultProps = {
-    hiddenFacets: {
-      layoutMode1: LAYOUT_MODE[0].value,
-      layoutMode2: LAYOUT_MODE[1].value,
+    mobileLayout: {
```

Comment:
Check if the [propTypes](https://github.com/vtex-apps/search-result/blob/master/react/constants/propTypes.js) are correct

Reply:
Fixed.